moped-editor/src/views/projects/projectView/ProjectFundingTable.js

```diff
@@ -547,7 +552,11 @@ const ProjectFundingTable = () => {
         data={data.moped_proj_funding}
         title={
           <div>
-            <Typography variant="h2" color="primary">
+            <Typography
+              variant="h2"
+              color="primary"
+              style={{ paddingTop: "1em", paddingBottom: "1em" }}
```

Comment:
I think only the `paddingTop` is needed, no extra `paddingBottom` unless the [mock](https://user-images.githubusercontent.com/1463708/150427836-a4e83698-cff0-470d-8512-14ea79a1b6b1.png) changed